cvelib/cli.py

```diff
@@ -298,32 +301,54 @@ def publish(ctx: click.Context, cve_id: str, cve_json_str: str, print_raw: bool)
 def reject(ctx: click.Context, cve_id: str, cve_json_str: str, print_raw: bool) -> None:
     """Reject a CVE record for a reserved or published CVE ID.
 
-    Will update if the CVE record already exists.
+    If the CVE is already rejected, this action will update its record if one is supplied.
+    A rejected CVE with a record can only be moved to the published state (see `cve publish`). A
+    rejected CVE without a record can be moved to the reserved state.
```

Comment:
This got split a little oddly, maybe it would look better if the whole second sentence was together on the last line:
```suggestion
    A rejected CVE without a record can be moved to the published or reserved states (see `cve publish` or `cve undo-reject`).
```

And maybe we should also explicitly say "You must supply a JSON record to reject a published CVE. Reserved CVEs can be rejected without a record.", either here or in README.md.